CMakeLists.txt

```diff
@@ -681,6 +681,23 @@ if (ZLIB_ENABLE_TESTS)
     target_link_libraries(example zlib)
     add_test(example example${CMAKE_EXECUTABLE_SUFFIX})
 
+    add_executable(compress_fuzzer test/fuzz/compress_fuzzer.c test/fuzz/standalone_fuzz_target_runner.c)
+    add_executable(example_small_fuzzer test/fuzz/example_small_fuzzer.c test/fuzz/standalone_fuzz_target_runner.c)
+    add_executable(example_large_fuzzer test/fuzz/example_large_fuzzer.c test/fuzz/standalone_fuzz_target_runner.c)
+    add_executable(example_flush_fuzzer test/fuzz/example_flush_fuzzer.c test/fuzz/standalone_fuzz_target_runner.c)
+    add_executable(example_dict_fuzzer test/fuzz/example_dict_fuzzer.c test/fuzz/standalone_fuzz_target_runner.c)
+    target_link_libraries(compress_fuzzer zlib)
+    target_link_libraries(example_small_fuzzer zlib)
+    target_link_libraries(example_large_fuzzer zlib)
+    target_link_libraries(example_flush_fuzzer zlib)
+    target_link_libraries(example_dict_fuzzer zlib)
+    file(GLOB ALL_SRC_FILES "${CMAKE_CURRENT_SOURCE_DIR}/*")
+    add_test(compress_fuzzer compress_fuzzer${CMAKE_EXECUTABLE_SUFFIX} ${ALL_SRC_FILES})
+    add_test(example_small_fuzzer example_small_fuzzer${CMAKE_EXECUTABLE_SUFFIX} ${ALL_SRC_FILES})
+    add_test(example_large_fuzzer example_large_fuzzer${CMAKE_EXECUTABLE_SUFFIX} ${ALL_SRC_FILES})
+    add_test(example_flush_fuzzer example_flush_fuzzer${CMAKE_EXECUTABLE_SUFFIX} ${ALL_SRC_FILES})
+    add_test(example_dict_fuzzer example_dict_fuzzer${CMAKE_EXECUTABLE_SUFFIX} ${ALL_SRC_FILES})
```

Comment:
cmake is missing a few sanitizers (crc, gz) that I added to configure+make.

cmake does not build the code of zlib-ng with sanitizers unless CFLAGS are set to contain the right flags.

Reply:
Like with configure, you could use an option to enable building and running the fuzzers.

Reply:
Fixed all these comments in https://github.com/sebpop/zlib-ng/commit/8215687ab29f74544c8af5540999db2e8dd03409
I will amend the patch with the fix.